### ISSUE TITLE
Square brackets don't work properly on WINDOWS with a german keyboard

### ISSUE THREAD

**Issue by aunkrig** [jline/jline2#269]
Hi there,

german keyboards (and probably other, non-US keyboards) require Ctrl+Alt for entering characters like "{[]}\". JLINE does not handle these keystrokes correctly, because "WindowsTerminal.java"

```
        // support some C1 control sequences: ALT + [@-_] (and [a-z]?) => ESC <ascii>
        // http://en.wikipedia.org/wiki/C0_and_C1_control_codes#C1_set
        int altState = KEY_EVENT_RECORD.LEFT_ALT_PRESSED | KEY_EVENT_RECORD.RIGHT_ALT_PRESSED;
        if (((keyEvent.uchar >= '@' && keyEvent.uchar <= '_') || (keyEvent.uchar >= 'a' && keyEvent.uchar <= 'z'))
            && (keyEvent.controlKeyState & altState) != 0) {
            sb.append('\u001B'); // ESC
        }
```
falsely identifies these keystrokes as "come C1 control sequences". I believe adding
```
&& (keyEvent.controlKeyState & (KEY_EVENT_RECORD.LEFT_CTRL_PRESSED | KEY_EVENT_RECORD.RIGHT_CTRL_PRESSED)) == 0
```
to the IF statement should do the trick.

Could you please include that change?

Regards,

Arno

(Observed with JLINE 2.12.)

**Comment by aunkrig** [comment]
Ah! The issue seems to be addressed in JLINE 3! What is the current status of JLINE3? Is 3.0.0 stable, or should I stay with JLINE2 for the time being?

**Comment by fantasy86** [comment]
already fixed since 2.12.1 ?  see #175

**Comment by aunkrig** [comment]
You're right; sorry for the hassle. I was using version 2.12, because the README.md told me so.
